### ISSUE TITLE
Mysql cache page has a 404 link.

### ISSUE THREAD

**Issue by alan-cugler** [saltstack/salt#56469]
relevant Salt docs URL: 
- https://docs.saltstack.com/en/latest/ref/cache/all/salt.cache.mysql_cache.html
relevant Docs build file: 
- ref/cache/all/salt.cache.mysql_cache
`(line   30) broken    http://python-mysql.readthedocs.io/en/latest/`

Comments:
- I manually verified 404 error.

**Comment by xeacott** [comment]
Yeah the documentation needs to be updated or at least find out why that URL is giving a 404 thanks for finding that. :)

**Comment by dwoz** [comment]
Fix merged into the source tree, pending release. Closing per bulk triage; will ship in the next release cycle. Reopen if this reproduces on a currently supported release once the fix is out.
